### PR TITLE
Clarifies pending/processing order options

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

The purpose of this PR is to clarify the available options in the button bar for `pending` and `processing` orders, and specifically explain why the `Cancel` button isn't always available.  Added screenshots and descriptions for each option with links to related topics.

## Affected documentation pages

[Updating Orders](https://docs.magento.com/m2/ee/user_guide/sales/order-update.html)
[Processing an Order](https://docs.magento.com/m2/ee/user_guide/sales/order-processing.html)

## Affected Magento editions

- [ x ] Open Source
- [ x ] Commerce
- [ x ] B2B

### CODE REVIEW

File: src/sales/order-update.md [hunk 1]
```diff
@@ -2,12 +2,30 @@
...
+
+<!--{% if "Default.CE Only" contains site.edition %}-->
+![]({% link images/images/order-button-bar-processing.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.EE-B2B Only" contains site.edition %}-->
+![]({% link images/images-ee/order-button-bar-processing.png %}){: .zoom}
+<!--{% endif %}-->
+_`Processing` Order Options_
+
+## Edit a pending order:
```
Comment (leslietilling) [inline src/sales/order-update.md]:
```suggestion
## Edit a pending order
```

Linting rule soon to be implemented will not allow punctuation at the end of a heading, so better to address these now as we see them.

File: src/sales/order-update.md [hunk 2]
```diff
@@ -21,13 +39,16 @@ _Edit Order_
...
+    _Edit Order_
+
 1. Apply your changes:
 
    - To save changes made to the billing or shipping address, click **Save**.
 
    - To save changes made to line items, and reprocess the order, click **Submit Order**.
 
-## To place an order on hold:
+## Place an order on hold:
```
Comment (leslietilling) [inline src/sales/order-update.md]: Same as above

```suggestion
## Place an order on hold
```

File: src/sales/order-update.md [hunk 3]
```diff
@@ -42,7 +63,7 @@ If the customer’s preferred method of payment is not available, or if the item
 
 To remove the hold on an order, edit the order. Then, click **Unhold**.
 
-## To cancel an order:
+## Cancel a pending order:
```
Comment (leslietilling) [inline src/sales/order-update.md]: Same as above

```suggestion
## Cancel a pending order
```

File: src/sales/order-update.md [hunk 4]
```diff
@@ -56,3 +77,29 @@ Canceling an order changes its status from `Pending` to `Canceled`.
     _Cancel Order_
 
 The status of the order is now `Canceled`.
+
+### Button Bar for Pending Orders
```
Comment (leslietilling) [inline src/sales/order-update.md]: If these are both H3, they are under the "Cancel a pending order" H2 (which doesn't really make sense). Maybe move each under the "Pending Orders" and "Processing Orders" H2s. 

Also, the `{: .buttons-table }` does not work unless you add the style declaration to the bottom of the page. Adjust the width of that first column according to what is needed to keep the button items from wrapping in the cell.

File: src/sales/order-update.md [hunk 5]
```diff
@@ -56,3 +77,29 @@ Canceling an order changes its status from `Pending` to `Canceled`.
...
+
+{: .buttons-table }
+|Button|Description|
+|--- |--- |
+|<span class="btn">Back</span>|Returns to the Orders page without saving changes.|
+|<span class="btn">Cancel</span>|Cancels the pending order.|
+|<span class="btn">Send Email</span>|Sends an email about the pending order to the customer.|
+|<span class="btn">Hold</span> / <span class="btn">Unhold</span>|Changes the status of the pending order to “On Hold”. To release the hold, choose “Unhold”.|
```
Comment (leslietilling) [inline src/sales/order-update.md]:
```suggestion
|<span class="btn">Hold</span> / <span class="btn">Unhold</span>|Changes the status of the pending order to _On Hold_. To release the hold, choose `Unhold`.|
```

File: src/sales/order-update.md [hunk 6]
```diff
@@ -56,3 +77,29 @@ Canceling an order changes its status from `Pending` to `Canceled`.
...
+
+### Button Bar for Processing Orders
+
+{: .buttons-table }
+|Button|Description|
+|--- |--- |
+|<span class="btn">Back</span>|Returns to the Orders page without saving changes.|
+|<span class="btn">Send Email</span>|Sends an email about the order to the customer.|
+|<span class="btn">Credit Memo</span>|Initiates the process to create a [credit memo]({% link sales/credit-memo-create.md %}).|
+|<span class="btn">Hold</span> / <span class="btn">Unhold</span>|Changes the status of the sales order to “On Hold”. To release the hold on the sales order, choose “Unhold”.|
```
Comment (leslietilling) [inline src/sales/order-update.md]:
```suggestion
|<span class="btn">Hold</span> / <span class="btn">Unhold</span>|Changes the status of the sales order to _On Hold_. To release the hold on the sales order, choose `Unhold`.|
```

File: src/sales/order-update.md (hunk 4 above)
Comment (ccondra) [reply src/sales/order-update.md]: Fixed. That's a good trick!

**Comment by ccondra** [comment]
running tests

**Pull request merged**